src/main/java/org/dependencytrack/event/kafka/processor/VulnerabilityScanResultProcessor.java

```diff
@@ -904,6 +927,24 @@ ON CONFLICT ("VULNERABILITY_ID", "COMPONENT_ID", "PROJECT_ID") DO UPDATE
                 """)
         void createAnalysisComments(@BindMethods final Iterable<AnalysisComment> comment);
 
+        @SqlQuery("""
+                UPDATE "ANALYSIS"
+                    SET "SUPPRESSED" = true, "STATE" = 'NOT_AFFECTED'
```

Comment:
Wondering if perhaps `FALSE_POSITIVE` would be a better state to assign. `NOT_AFFECTED` still suggests that the vulnerability itself is valid, which in this case it wouldn't be.